felix/bpf/conntrack/cttestdata/ct_data.go

```diff
@@ -135,6 +136,31 @@ func init() {
 			ExpectedDeletions: []conntrack.Key{tcpFwdKey, tcpRevKey},
 		},
 
+		CTCleanupTest{
+			Description: "long-lived TCP NAT entries (DSR)",
+			KVs: map[conntrack.Key]conntrack.Value{
+				// Note: last seen time on the forward entry should be ignored in
+				// favour of the last-seen time on the reverse entry.
+				tcpFwdKey: conntrack.NewValueNATForward(Now-3*time.Hour, 0, tcpRevKey),
+				tcpRevKey: conntrack.NewValueNATReverse(Now-59*time.Minute, v3.FlagNATFwdDsr,
+					conntrack.Leg{SynSeen: true, AckSeen: true}, conntrack.Leg{SynSeen: false, AckSeen: false},
+					nil, nil, 5555),
+			},
+		},
+
+		CTCleanupTest{
+			Description: "expired TCP NAT entries (DSR)",
+			KVs: map[conntrack.Key]conntrack.Value{
+				// Note: last seen time on the forward entry should be ignored in
+				// favour of the last-seen time on the reverse entry.
+				tcpFwdKey: conntrack.NewValueNATForward(Now-3*time.Hour, 0, tcpRevKey),
+				tcpRevKey: conntrack.NewValueNATReverse(Now-2*time.Hour, 0,
```

Comment:
is this test missing the dsr flag. Title says DSR

Reply:
Looks like it